### PR TITLE
Issue #216

### PR DESCRIPTION
**Pull request by romulocollopy**
Now m2m related objects are created if flag `make_m2m=True` is passed together with `_fill_optional=True` or `_fill_optional="name_of_field"`

### CODE REVIEW

File: test/generic/tests/test_mommy.py [hunk 1]
```diff
@@ -350,6 +350,24 @@ def test_skip_null(self):
...
+
+    def test_create_nullable_many_to_many_if_flagged_and_fill_optional(self):
+        classroom = mommy.make(Classroom, make_m2m=True, _fill_optional=True)
+        self.assertEqual(classroom.students.count(), 5)
+
+    def test_nullable_many_to_many_is_not_created_if_not_flagged_and_fill_optional(self):
+        classroom = mommy.make(Classroom, make_m2m=False, _fill_optional=True)
+        self.assertEqual(classroom.students.count(), 0)
+
+    # test_nullable_many_to_many_is_not_created_even_if_flagged_and_not_fill_optional in line 252
```
Comment (berinhard) [inline test/generic/tests/test_mommy.py]: I'll just remove this comment because it doesn't make to much sense.

Comment (romulocollopy) [inline test/generic/tests/test_mommy.py]: done.

**Comment by vandersonmota** [comment]
thank you @romulocollopy

**Comment by romulocollopy** [comment]
Be my guest @vandersonmota

**Pull request merged**